gcp/README.md

```diff
@@ -0,0 +1,194 @@
+# K8ssandra GCP Terraform Example
+
+## GCP resources
+* GKE cluster
+* Cluster node pool
+* service account 
+* iam member 
+* custom iam member
+* google compute network(VPC)
+* public subnet
+* private subnet
+* router
+* compute router nat
+* google storage bucket
+* google storage bucket iam member
+* google compute address
+
+
+## Project directory Structure
+<pre>
+gcp/
+ ├──modules/
+ |  ├──<a href="modules/gcs/README.md">gcs</a>
+ |     ├── main.tf 
+ |     └── variables.tf 
+ |     └── outputs.tf 
+ |     └── README.md 
+ |  ├──<a href="modules/vpc/README.md">vpc</a>
+ |     ├── main.tf 
+ |     └── variables.tf 
+ |     └── outputs.tf 
+ |     └── README.md 
+ |  ├──<a href="modules/iam/README.md">iam</a>
+ |     ├── main.tf 
+ |     └── variables.tf 
+ |     └── outputs.tf 
+ |     └── README.md
+ |  ├──<a href="modules/gke/README.md">gke</a>
+ |     ├── main.tf 
+ |     └── variables.tf 
+ |     └── outputs.tf 
+ |     └── README.md
+ └──README.md
+ └──gitignore
+ |
+ ├──<a href="env/README.md">env</a>
+    ├── dev.tf
+      ../modules/vpc
+      ../modules/iam
+      ../modules/gke
+      ../modules/gcs
+    ├── version.tf 
+    └── backend.tf 
+    └── variables.tf 
+    └── outputs.tf
+    └── README.md
+</pre>
+
+## Prerequisites
+
+|       NAME        |   Version  | 
+|-------------------|------------|
+| terraform version |   0.14     |
+| gcp provider      |   ~>3.0    |
+| Helm version      |   v3.5.3   |
+| Google Cloud SDK  |   333.0.0  |
+|    bq             |   2.0.65   |
+|   core            | 2021.03.19 |
+|  gsutil           |    4.60    |
+|  kubectl          |  1.17.17   |
+
+The steps to create kubernetes cluster in this document require the following tools installation and configuration to access Google cloud resources.
+
+### Cloud project
+
+You will need a google cloud project created, If you do not have a google cloud account plese sigup [google](https://cloud.google.com).  
+
+### Required GCP APIs
+
+The following APIs will be enabled:
+
+* Compute Engine API
+* Kubernetes Engine API
+* Cloud SQL Admin API
+* Secret Token API
+* Stackdriver Logging API
+* Stackdriver Monitoring API
+* IAM Service Account Credentials API
+
+All the tools for the demo are installed. When using Cloud Shell execute the following command in order to setup gcloud cli. When executing this command please setup your region and zone.
```

Comment:
> All the tools for the demo are installed

I do not understand what you are saying.

> When using Cloud Shell execute the following command in order to setup gcloud cli. When executing this command please setup your region and zone.

Let's not mention Cloud Shell.  Write the instructions so that it works on Linux, and it is tested on Linux.